playbooks/prepare_host.yaml

```diff
@@ -69,3 +69,39 @@
     user:
       name: stack
       groups: wheel
+  - name: Prepare host for Ceph
+    when: ceph_enabled
+    block:
+    - name: Make sure we have losetup installed/latest
+      package:
+        name:
+          - util-linux
+          - lvm2
+        state: latest
+    - name: Use dd and losetup to create the loop devices
+      shell: |
+        dd if=/dev/zero of=/var/lib/ceph-osd.img bs=1 count=0 seek={{ ceph_loop_device_size }}G
+        losetup /dev/loop3 /var/lib/ceph-osd.img
```

Comment:
If we're naming this loop device explicitly, might be better to be clearer and avoid potential conflicts by calling it something like `/dev/ceph-loopback`?